docs/fiddles/native-ui/drag-and-drop/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Drag and drop files</title>
+  </head>
+
+  <body>
+    <div class="section-wrapper">
+      <h1>Drag and drop files</h1>
+
+      <h3>
+        Electron supports dragging files and content out from web content into
+        the operating system's world.
+      </h3>
+
+      <p>
+        Open the
+        <a href="https://electronjs.org/docs/tutorial/native-file-drag-drop"
+          >full API documentation<span class="u-visible-to-screen-reader"
+            >(opens in new window)</span

```

Comment:
Due to `u-visible-to-screen-reader` not being too useful here, I'd remove the `<span>` wrapping this.

Reply:
done